zstackctl/zstackctl/migrate_influxdb_data.py

```
@@ -813,18 +813,23 @@ def print_blue(strs):
     def __init__(self, key='ZStack open source'):
         self.key = md5(key.encode()).hexdigest()
-        self.cipher = AES.new(self.key, AES.MODE_ECB)
+        self.cipher = AES.new(self.key.encode(), AES.MODE_ECB)
         self.prefix = "crypt_key_for_v1::"
         self.BLOCK_SIZE = 16
 
     # PKCS#7
-    def _pad(self, data_to_pad, block_size):
+    def _pad(self, data_to_pad: str, block_size: int) -> bytes:
         padding_len = block_size - len(data_to_pad) % block_size
         padding = bchr(padding_len) * padding_len
```

Ruff (0.11.9)

`bchr` may be undefined, or defined from star imports

(F405)